pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -888,10 +888,10 @@ public void deletePartitionedTopic(
             @ApiParam(value = "Is authentication required to perform this operation")
             @QueryParam("authoritative") @DefaultValue("false") boolean authoritative,
             @ApiParam(value = "Delete the topic's schema storage")
-            @QueryParam("deleteSchema") @DefaultValue("false") boolean deleteSchema) {
+            @QueryParam("deleteSchema") @DefaultValue("true") boolean deleteSchema) {
```

Comment:
We can remove it from the API definition?

Reply:
All done